index.html

```diff
@@ -54,13 +68,13 @@ <h2>Projects</h2>
               <li>JavaScript</li>
             </ul>
             <div class="project-butt">
-              <p>See this project</p>
+              <p class="text-butt">See this project</p>
```

Comment:
- [ ] **Good job done so far** 👏. Please implement the hover effect styles for all your buttons. Remember it is important that you try to replicate the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A553) as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

you can Kindly follow the given button interaction styling on all your buttons :joy:
![image](https://user-images.githubusercontent.com/102757126/199978566-c4753bb1-e7cc-4872-bb53-03f8706d4962.png)

  - Please I would recommend replacing this ```p``` tag with a ```button``` or an ```anchor``` as the ```<a>``` tag defines a hyperlink, which is used to link from one page to another and also can be used to download embedded files on your webpage, 👍 👍 
  - Kindly try to add the allow icon on this button as given on Figma design.
   
**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/199981672-ee26539a-1104-43ba-bac5-24ab04a2a444.png) | ![image](https://user-images.githubusercontent.com/102757126/199981775-6c779f47-546c-4d23-ab15-182c274b762b.png)